src/pages/CategoriesPage/AddCategory/AddNewCategoryComponent.tsx

```diff
@@ -7,73 +7,86 @@ import {
+import { CreateUpdateCategoryModel } from '../../../models/categories/CreateCategoryModel';
+import { validationSchema } from '../../../validationSchemas/validationSchema';
 import { styles } from '../styles';
 
 interface Props {
   createModalOpen: boolean;
-  onSubmit: (values: CreateCategoryModel) => void;
+  onSubmit: (values: CreateUpdateCategoryModel) => void;
   onToggleCreateStatusClick: () => void;
 }
 
 const AddNewCategoryComponent = (props: Props) => {
   return (
     <Dialog open={props.createModalOpen}>
-      <DialogTitle sx={styles.dialogTitle}>CREATE CATEGORY</DialogTitle>
+      <div style={styles.dialogTitleWrapper}>
+        <DialogTitle sx={styles.dialogTitle}>Click on the confirmation button to create the category</DialogTitle>
+      </div>
       <DialogContent>
-        <Formik initialValues={{
-          name: '',
-          description: '',
-        }} validationSchema = {createValidationSchema} onSubmit = {(values: CreateCategoryModel) => { props.onSubmit(values)}}>
-          { formProps => (<form onSubmit={formProps.handleSubmit}>
-            <div style={styles.fieldsWrapper}>
-              <TextField
-                id="name"
-                label="Category Name"
-                variant="outlined"
-                onChange={formProps.handleChange}
-                value={formProps.values.name}
-                error={formProps.touched.name && Boolean(formProps.errors.name)}
-                helperText={formProps.touched.name && formProps.errors.name}
-              />
-            </div>
-            <div style={styles.fieldsWrapper}>
-              <TextField
-                id="description"
-                label="Description"
-                variant="outlined"
-                onChange={formProps.handleChange}
-                value={formProps.values.description}
-                error={
-                  formProps.touched.description &&
-                  Boolean(formProps.errors.description)
-                }
-                helperText={
-                  formProps.touched.description && formProps.errors.description
-                }
-              />
-            </div>
-            <Grid container>
-              <Grid item xs={6}>
-                <div style={styles.buttonsWrapper}>
-                  <Button
-                    sx={styles.closeButton}
-                    onClick={props.onToggleCreateStatusClick}
-                  >
-                    Close
-                  </Button>
-                </div>
+        <Formik
+          initialValues={{
+            name: '',
+            description: '',
+          }}
+          validationSchema={validationSchema}
+          onSubmit={(values: CreateUpdateCategoryModel) => {
+            props.onSubmit(values);
+          }}
+        >
+          {(formProps) => (
+            <form onSubmit={formProps.handleSubmit}>
+              <div style={styles.fieldsWrapper}>
```

Comment:
Do not use DIV. Use Box instead.

Reply:
All divs in the modal windows have been replaced by the Box components